cmd/tendermint/commands/root.go

```diff
@@ -0,0 +1,76 @@
+package commands
+
+import (
+	"github.com/spf13/cobra"
+
+	cfg "github.com/tendermint/go-config"
+	"github.com/tendermint/go-logger"
+	tmcfg "github.com/tendermint/tendermint/config/tendermint"
+)
+
+var config cfg.Config
+var log = logger.New("module", "main")
+
+//global flags
+var (
+	printHelp     bool
+	moniker       string
+	nodeLaddr     string
+	seeds         string
+	fastSync      bool
+	skipUPNP      bool
+	rpcLaddr      string
+	grpcLaddr     string
+	logLevel      string
+	proxyApp      string
+	abciTransport string
+
+	pex bool
+)
+
+var RootCmd = &cobra.Command{
+	Use:   "tendermint",
+	Short: "Tendermint Core (BFT Consensus) in Go",
+}
+
+func init() {
+
+	// Get configuration
+	config = tmcfg.GetConfig("")
+
+	/////////////////////
+	// parse flags
+
+	// configuration options
+	RootCmd.PersistentFlags().StringVar(&moniker, "moniker", config.GetString("moniker"), "Node Name")
```

Comment:
Correct me if I am wrong, but all these flags belong to `runNodeCmd` and should be moved there. Except maybe `log_level`. Can we do that?